sdcm/cluster.py

```diff
@@ -1908,7 +1908,6 @@ def update_repo_cache(self):
                 # try to avoid ERROR 404 of yum, reference https://wiki.centos.org/yum-errors
```

Comment:
@fgelcer , i thought of removing this comment, but on second thought, it might be usefull to leave it for a while, to verify no flakiness or other issues pops up in coming future.